alteruserscramcredentials_test.go

```diff
@@ -0,0 +1,80 @@
+package kafka
+
+import (
+	"context"
+	"testing"
+
+	ktesting "github.com/segmentio/kafka-go/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAlterUserScramCredentials(t *testing.T) {
```

Comment:
Would it be of value to test deletions as well?

Reply:
Good idea, just added them